exploration/admin_lookup.md

```diff
@@ -56,23 +56,26 @@ with tempfile.TemporaryDirectory() as temp_dir:
         iso3 = row["iso3"]
         max_adm_level = row["max_adm_level"]
         load_shp_from_azure(iso3, temp_dir, MODE)
-        gdf = gpd.read_file(f"{temp_dir}/{iso3.lower()}_adm{max_adm_level}.shp")
-
-        # Get name column and its language code
-        name_column = select_name_column(gdf, max_adm_level)
-        language_code = name_column[-2:]
-        name_columns = [f"ADM{i}_{language_code}" for i in range(0, max_adm_level + 1)]
-
-        # Standardize column names and add language info
-        new_columns = [x.replace(f"_{language_code}", "_NAME") for x in name_columns]
-        gdf = gdf.rename(columns=dict(zip(name_columns, new_columns)))
-        gdf["NAME_LANGUAGE"] = language_code
-        gdf["ISO3"] = iso3
-        gdf["ADM_LEVEL"] = max_adm_level
-
-        # Keep only relevant columns
-        matching_cols = [col for col in gdf.columns if col in DEFAULT_COLS]
-        dfs.append(gdf[matching_cols])
+
+        name_columns = []
+        for admin_level in range(max_adm_level + 1):
+            gdf = gpd.read_file(f"{temp_dir}/{iso3.lower()}_adm{admin_level}.shp")
+
+            # Get name column and its language code
+            name_column = select_name_column(gdf, admin_level)
+            language_code = name_column[-2:]
+            name_columns.append(name_column)
+
+            # Standardize column names and add language info
+            new_columns = [x.replace(f"_{language_code}", "_NAME") for x in name_columns]
+            gdf = gdf.rename(columns=dict(zip(name_columns, new_columns)))
+            gdf["NAME_LANGUAGE"] = language_code
+            gdf["ISO3"] = iso3
+            gdf["ADM_LEVEL"] = max_adm_level
```

Comment:
this part seems add to me. I don't think we should list the `max_adm_level` as the `ADM_LEVEL`. I think `ADM_LEVEL` should reflect the admin level of the data on that row (i.e `admin_level`). 

Current set up results in data that looks like this:

```
# A tibble: 476 × 9
   adm0_name   adm0_pcode name_language iso3  adm_level adm1_name  adm1_pcode adm2_name adm2_pcode
   <chr>       <chr>      <chr>         <chr>     <int> <chr>      <chr>      <chr>     <chr>     
 1 Afghanistan AF         EN            AFG           2 NA         NA         NA        NA        
 2 Afghanistan AF         EN            AFG           2 Nimroz     AF34       NA        NA        
 3 Afghanistan AF         EN            AFG           2 Kunar      AF15       NA        NA 
 ```
where  it says `ADM_LEVEL = 2`, but is NA for adm2 attributes... We could include a `MAX_ADM_LEVEL` col if there is a reason you want to keep it, but i'd vote not to have it in this table as it's easy enough and intuitive for a user to calculate